src/node_buffer.cc

```diff
@@ -949,6 +983,8 @@ void IndexOfBuffer(const FunctionCallbackInfo<Value>& args) {
   THROW_AND_RETURN_UNLESS_BUFFER(Environment::GetCurrent(args), args[0]);
   SPREAD_ARG(args[0], ts_obj);
   SPREAD_ARG(args[1], buf);
+  int64_t offset_i64 = args[2]->IntegerValue();
+  bool is_forward = args[4]->BooleanValue();
 
   if (buf_length > 0)
     CHECK_NE(buf_data, nullptr);
```

Comment:
This is an artifact of old code. `SPREAD_ARG` already does this check. Feel free to remove these checks in applicable methods.